scl/scl.conf

```diff
@@ -25,6 +25,16 @@
 # `include-path`, then includes all SCL supplied plugins.
 #
 
-@include 'scl/*/*.conf'
-
 @define java-module-dir "`module-install-dir`/java-modules"
+
+@requires json-plugin \
```

Comment:
The `@require` initially have been done due to json plugin not available all the time. (https://github.com/syslog-ng/syslog-ng/issues/673).

I would first of all split this part, as the requires message work is fine. This part where you make json mandatory is kinda a different issue. (If we want to do that I would also make it not optional to compile json module, as in this way it is really easy to make it fail.)

Reply:
I have now moved all of this into a separate patch that can be split off to a separate branch if needed, I just didn't want to add the detailed comment on the missing module into n+1 locations.

like I said in the other thread, at this point I would opt for making json-plugin mandatory for the SCL. IF someone doesn't need the functionality of the SCL they can forgo it simply by removing the "@include scl.conf" in their configuration.